lib/GridItem.jsx

```diff
@@ -415,6 +458,106 @@ export default class GridItem extends React.Component<Props, State> {
     );
   }
 
+  delayEvents = React.createRef();
```

Comment:
This doesn't need to be a ref, it can just be a plain array.

Reply:
Fair. I'm used to creating refs for working with functional components but there's no need for them here.